eng/pipelines/libraries/outerloop.yml

```diff
@@ -14,6 +14,29 @@ variables:
 jobs:
   - template: /eng/pipelines/common/checkout-job.yml
   
+  #
+  # CoreLib Build
+  #
+  - template: /eng/pipelines/common/platform-matrix.yml
```

Comment:
To simplify this, we should be able to use `platform-matrix-multijob` on this yml. Not a blocker, I'll leave it up to you.

Reply:
will defer to next PR.

Reply:
Sounds good.